src/index.css

```
@@ -264,4 +264,30 @@ body {
 }
 ::-webkit-scrollbar-thumb:active {
   background-color: #676766;
+}
+
+/************************************************/
+/*             FOOTER COMPONENT                 */
+/************************************************/
+
+.custom-underline {
+  position: relative;
+}
+
+.custom-underline::before {
+  position: absolute;
+  width: 100%;
+  height: 2px;
+  right: 0;
+  bottom: 0;
+  content: '';
+  left: 0;
+  border-radius: 30%;
+  transform: scaleX(0);
+  background-color: #fff;
+  transition: transform 0.3s ease-out;
+}
```

```suggestion
.custom-underline::before {
  position: absolute;
  width: 100%;
  height: 2px;
  inset-inline: 0;
  bottom: 0;
  content: '';
  border-radius: 30%;
  transform: scaleX(0);
  background-color: #fff;
  transition: transform 0.3s ease-out;
}
```

Solo para acortar un poco y aprendan un poco de operadores

Referencia de [Operadores Logicos](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values)